pixelmatch/__init__.py

```diff
@@ -223,19 +235,31 @@ def rgb2q(r: int, g: int, b: int):
     return r * 0.21147017 - g * 0.52261711 + b * 0.31114694
 
 
-def blend(c, a):
+def blendRGB(r: int, g: int, b: int, a: int):
+    """
+    Blend r, g, and b with a
+    :param r: red channel to blend with a
+    :param g: green channel to blend with a
+    :param b: blue channel to blend with a
+    :param a: alpha to blend with
+    :return: tuple of blended r, g, b
+    """
+    return blend(r, a), blend(g, a), blend(b, a)
+
+
+def blend(c: float, a: int):
     """blend semi-transparent color with white"""
     return 255 + (c - 255) * a
 
 
-def draw_pixel(output, pos: int, r: int, g: int, b: int):
+def draw_pixel(output: MutableImageSequence, pos: int, r: int, g: int, b: int):
```

Comment:
`MutableImageSequence` is typed as `MutableSequence[float]`: https://github.com/whtsky/pixelmatch-py/pull/38/files#diff-c9a626a30caa55ab4ae697edf9b3e74aR4

But we are writing int into it..

---

Sadly users can't pass a `MutableSeuqnce[int]` here.

![图片](https://user-images.githubusercontent.com/876694/81426217-78b79400-918b-11ea-91cd-24e74ca0acb9.png)

Reply:
I have to say get types hints done right in Python is pretty hard, and that's why I didn't add type hints for image data at first.

Reply:
From [PEP 484](https://www.python.org/dev/peps/pep-0484/):

> Rather than requiring that users write import numbers and then use numbers.Float etc., this PEP proposes a straightforward shortcut that is almost as effective: when an argument is annotated as having type float, an argument of type int is acceptable; similar, for an argument annotated as having type complex, arguments of type float or int are acceptable.

It looks like you are using Pyright. This is a known issue that has been fixed, though: https://github.com/microsoft/pyright/issues/260

Maybe your Pyright version is out of date?

Reply:
I'm using VSCode with [Pyright](https://marketplace.visualstudio.com/items?itemName=ms-pyright.pyright) extension v1.1.35

Reply:
I can reproduce this with mypy 0.770.
```python
from typing import MutableSequence

a: MutableSequence[int] = [255, 255, 0]
aa_color: MutableSequence[float] = a
```
![图片](https://user-images.githubusercontent.com/876694/81428525-ea451180-918e-11ea-83cc-13ce02a39be7.png)


( Sorry for the color scheme, I was just too lazy to change them in VSCode)

Reply:
lol. On the surface it seems like a bug, but since it exists in both pyright and mypy I'll just work around it.

Reply:
I added mypy in #41 . This should make things a bit easier.